### DISCUSSION TITLE
Official website loading speed

### DISCUSSION THREAD

**Discussion by hzjsea** [fastapi/fastapi#7555] [Questions] [answered]
I think a large portion of learners are rejected by the speed of the official documentation website

Since it is a web framework, can it increase the speed of the website, **Fastapi** is an excellent framework, more people should learn it, should it increase the loading speed, thanks

**Comment by phy25** [comment]
I am not sure what you mean by slow. Could you provide more information on which part is slow? How slow it is? Probably a screenshot of Web Developers network tab would help.

**Comment by hzjsea** [comment]
> 我不确定你的意思是慢。您能否提供有关哪个部分运行缓慢的更多信息？有多慢？Web Developers网络选项卡的屏幕快照可能会有所帮助。


I found him to be intermittent, sometimes very slow, I was watching for a while
But I ’m sure it ’s not my network problem

**Comment by phy25** [comment]
Where are you? What network provider do you use?

**Comment by hzjsea** [comment]
> 你在哪？您使用什么网络提供商？


I can't see anything from the network
![image](https://user-images.githubusercontent.com/61597549/80284307-4f8d1180-8750-11ea-89cc-feaf2310b9ef.png)
![image](https://user-images.githubusercontent.com/61597549/80284351-a7c41380-8750-11ea-96dd-7dac5efb7113.png)

I'm from China, but I can visit websites like youtube, and it's very fast

**Comment by tiangolo** [comment, marked as answer]
The docs website is served by Netlify, which is the same that serves things like Vue.js docs. So it's probably not the bottleneck, maybe there's a network issue on your side, maybe the ISP is throttling bandwidth or a firewall limiting connections.

I don't see why they could target FastAPI, there's no "controversial" content nor heavy files (like videos), but that's still more probable.

Anyway, I imagine you solved your problem, thanks for closing the issue. :+1:
